spec/models/blood_pressure_spec.rb

```diff
@@ -15,6 +15,21 @@
     it_behaves_like "a record that is deletable"
   end
 
+  it "has a valid encounter / observation for bp_with_encounter factory" do
+    user = create(:user)
+    patient = create(:patient)
+    bp = create(:bp_with_encounter, user: user, patient: patient)
+    bp.reload
```

Comment:
Why would a reload be needed on a new object?

Reply:
Just being sure here, sometimes associations (especially from fixtures) can act weirdly for cases like this.